app/models/order.rb

```diff
@@ -0,0 +1,13 @@
+class Order < ActiveRecord::Base
+  belongs_to :user
+  has_many :order_staches
+  has_many :staches, through: :order_staches
+
+  def total
+    staches.map(&:subtotal_num).sum
+  end
+
+  def completed?
+    %w(completed cancelled).include?(status) ? "Yes" : "No"
```

Comment:
I know this method is to fill in part of the views for orders but normally methods ending in a question mark return a boolean - what are your thoughts on changing this method to return a boolean or changing the call to it?